client/core/bookie.go

```diff
@@ -511,6 +511,8 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 			tracker.metaData.Host == dc.acct.host && tracker.metaData.Status == order.OrderStatusBooked {
 			// Locally revoke the purged book order.
 			tracker.revoke()
+			details := fmt.Sprintf("Order %v revoked.", tracker.Order.ID())
+			c.notify(newOrderNote("Order Revoked", details, db.ErrorLevel, tracker.coreOrderInternal()))
```

Comment:
Not sure we want to have this one be an error since it's a scheduled event. For `handleRevokeOrderMsg`, I'd agree error is best there.  Maybe warning? @buck54321?

Reply:
Buck indicated it should be warning, it should have been that way already. The test missed it because it was not properly checking. It's fixed now.